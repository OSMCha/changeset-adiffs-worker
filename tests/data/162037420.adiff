<?xml version="1.0"?>
<osm version="0.6" generator="osmcha" copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
  <action type="modify">
    <old>
      <node id="115839045" version="2" lon="-91.84139" lat="32.916038"/>
    </old>
    <new>
      <node id="115839045" version="3" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" lat="32.9161344" lon="-91.8415109"/>
    </new>
  </action>
  <action type="delete">
    <old>
      <node id="115868091" version="2" lon="-91.849971" lat="32.912256"/>
    </old>
    <new>
      <node id="115868091" version="3" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" lat="32.912256" lon="-91.849971" visible="false"/>
    </new>
  </action>
  <action type="modify">
    <old>
      <node id="115868096" version="2" lon="-91.842096" lat="32.91558"/>
    </old>
    <new>
      <node id="115868096" version="3" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" lat="32.9155243" lon="-91.8425037"/>
    </new>
  </action>
  <action type="delete">
    <old>
      <node id="115868099" version="2" lon="-91.841579" lat="32.915915"/>
    </old>
    <new>
      <node id="115868099" version="3" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" lat="32.915915" lon="-91.841579" visible="false"/>
    </new>
  </action>
  <action type="create">
    <node id="12550908584" version="1" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" lat="32.9157457" lon="-91.8420375"/>
  </action>
  <action type="create">
    <node id="12550908585" version="1" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" lat="32.9159882" lon="-91.8416835"/>
  </action>
  <action type="modify">
    <old>
      <way id="12683351" version="2" user="bot-mode" uid="451693" timestamp="2013-02-08T12:16:09Z" changeset="14955159">
        <bounds minlat="32.914596" minlon="-91.84139" maxlat="32.916038" maxlon="-91.83851"/>
        <nd ref="115839038" lon="-91.83851" lat="32.914596"/>
        <nd ref="115839040" lon="-91.839772" lat="32.915228"/>
        <nd ref="115839043" lon="-91.841142" lat="32.915914"/>
        <nd ref="115839045" lon="-91.84139" lat="32.916038"/>
        <tag k="highway" v="residential"/>
        <tag k="name" v="Sawyer Road"/>
        <tag k="tiger:cfcc" v="A41"/>
        <tag k="tiger:county" v="Morehouse, LA"/>
        <tag k="tiger:name_base" v="Sawyer"/>
        <tag k="tiger:name_type" v="Rd"/>
        <tag k="tiger:reviewed" v="no"/>
        <tag k="tiger:zip_left" v="71220"/>
        <tag k="tiger:zip_right" v="71220"/>
      </way>
    </old>
    <new>
      <way id="12683351" version="2" user="bot-mode" uid="451693" timestamp="2013-02-08T12:16:09Z" changeset="14955159">
        <nd ref="115839038" lon="-91.83851" lat="32.914596"/>
        <nd ref="115839040" lon="-91.839772" lat="32.915228"/>
        <nd ref="115839043" lon="-91.841142" lat="32.915914"/>
        <nd ref="115839045" lon="-91.8415109" lat="32.9161344"/>
        <tag k="highway" v="residential"/>
        <tag k="name" v="Sawyer Road"/>
        <tag k="tiger:cfcc" v="A41"/>
        <tag k="tiger:county" v="Morehouse, LA"/>
        <tag k="tiger:name_base" v="Sawyer"/>
        <tag k="tiger:name_type" v="Rd"/>
        <tag k="tiger:reviewed" v="no"/>
        <tag k="tiger:zip_left" v="71220"/>
        <tag k="tiger:zip_right" v="71220"/>
      </way>
    </new>
  </action>
  <action type="modify">
    <old>
      <way id="38476787" version="7" user="UP508" uid="9503506" timestamp="2024-08-02T13:39:04Z" changeset="154737304">
        <bounds minlat="32.9020186" minlon="-91.861496" maxlat="32.92824" maxlon="-91.8207085"/>
        <nd ref="454857545" lon="-91.8443966" lat="32.9020186"/>
        <nd ref="454857543" lon="-91.8448456" lat="32.9029955"/>
        <nd ref="454857544" lon="-91.8454046" lat="32.9035615"/>
        <nd ref="454857558" lon="-91.8468446" lat="32.9045575"/>
        <nd ref="454857559" lon="-91.8484107" lat="32.9049405"/>
        <nd ref="454857556" lon="-91.8504527" lat="32.9051045"/>
        <nd ref="454857557" lon="-91.8526598" lat="32.9046215"/>
        <nd ref="454857554" lon="-91.8539718" lat="32.9032796"/>
        <nd ref="454857555" lon="-91.8543438" lat="32.9026796"/>
        <nd ref="454857549" lon="-91.861496" lat="32.9027786"/>
        <nd ref="454857553" lon="-91.860905" lat="32.9054005"/>
        <nd ref="454857173" lon="-91.8561459" lat="32.9054995"/>
        <nd ref="454857184" lon="-91.8512167" lat="32.9054401"/>
        <nd ref="8643612207" lon="-91.8511352" lat="32.911748"/>
        <nd ref="8663693250" lon="-91.8511344" lat="32.9118078"/>
        <nd ref="8663693217" lon="-91.8511317" lat="32.9120205"/>
        <nd ref="8643612206" lon="-91.8511293" lat="32.9122029"/>
        <nd ref="454857160" lon="-91.8510753" lat="32.9163836"/>
        <nd ref="8663693232" lon="-91.8414184" lat="32.9165001"/>
        <nd ref="8663693255" lon="-91.8411104" lat="32.9165038"/>
        <nd ref="454857285" lon="-91.8399576" lat="32.9165177"/>
        <nd ref="8663693251" lon="-91.8399464" lat="32.9153915"/>
        <nd ref="8663693256" lon="-91.8399447" lat="32.9152154"/>
        <nd ref="454857292" lon="-91.8398865" lat="32.9093633"/>
        <nd ref="454857197" lon="-91.8341179" lat="32.9093928"/>
        <nd ref="454857275" lon="-91.8340808" lat="32.9165578"/>
        <nd ref="454857129" lon="-91.8338187" lat="32.9238093"/>
        <nd ref="454857135" lon="-91.8353657" lat="32.9238119"/>
        <nd ref="454857109" lon="-91.8349009" lat="32.9242902"/>
        <nd ref="454857117" lon="-91.8347323" lat="32.9246368"/>
        <nd ref="454857150" lon="-91.834218" lat="32.9251418"/>
        <nd ref="454857151" lon="-91.8341129" lat="32.9255159"/>
        <nd ref="454857147" lon="-91.833249" lat="32.9267538"/>
        <nd ref="454857148" lon="-91.832443" lat="32.9270364"/>
        <nd ref="454857090" lon="-91.8319378" lat="32.9267825"/>
        <nd ref="454857094" lon="-91.8314162" lat="32.9268389"/>
        <nd ref="454857332" lon="-91.8300767" lat="32.9274146"/>
        <nd ref="454857322" lon="-91.8292975" lat="32.9275448"/>
        <nd ref="454857295" lon="-91.8285996" lat="32.9275833"/>
        <nd ref="454857293" lon="-91.8255075" lat="32.9276153"/>
        <nd ref="454857316" lon="-91.8249224" lat="32.9276149"/>
        <nd ref="454857476" lon="-91.8243566" lat="32.9277079"/>
        <nd ref="454857480" lon="-91.8231934" lat="32.92824"/>
        <nd ref="454857479" lon="-91.8229331" lat="32.9279145"/>
        <nd ref="454857482" lon="-91.8222509" lat="32.9273068"/>
        <nd ref="454857481" lon="-91.8220956" lat="32.9270667"/>
        <nd ref="454857483" lon="-91.8210908" lat="32.9251771"/>
        <nd ref="454857485" lon="-91.820938" lat="32.9247557"/>
        <nd ref="454857343" lon="-91.820805" lat="32.9245445"/>
        <nd ref="454857351" lon="-91.8207085" lat="32.9241427"/>
        <nd ref="454857387" lon="-91.8207496" lat="32.9237505"/>
        <nd ref="454857368" lon="-91.8208294" lat="32.9236686"/>
        <nd ref="454857403" lon="-91.8252572" lat="32.9236756"/>
        <nd ref="454857421" lon="-91.8252823" lat="32.9226449"/>
        <nd ref="454857444" lon="-91.8256608" lat="32.9226625"/>
        <nd ref="454857435" lon="-91.8258778" lat="32.9226137"/>
        <nd ref="454857457" lon="-91.8263411" lat="32.9223782"/>
        <nd ref="454857470" lon="-91.8271573" lat="32.9214707"/>
        <nd ref="454857471" lon="-91.8278228" lat="32.9208158"/>
        <nd ref="454857472" lon="-91.8279871" lat="32.9205835"/>
        <nd ref="454857473" lon="-91.8281948" lat="32.9198528"/>
        <nd ref="454857474" lon="-91.8281165" lat="32.9194017"/>
        <nd ref="454857475" lon="-91.827807" lat="32.9190067"/>
        <nd ref="454857532" lon="-91.8270937" lat="32.9185523"/>
        <nd ref="454857530" lon="-91.8267751" lat="32.9182978"/>
        <nd ref="454857535" lon="-91.8264526" lat="32.9178034"/>
        <nd ref="454857534" lon="-91.8264445" lat="32.917448"/>
        <nd ref="454857533" lon="-91.826539" lat="32.9171047"/>
        <nd ref="454857539" lon="-91.8265842" lat="32.9165342"/>
        <nd ref="454857538" lon="-91.8262383" lat="32.9156588"/>
        <nd ref="454857542" lon="-91.8262413" lat="32.9154327"/>
        <nd ref="454857541" lon="-91.8264226" lat="32.914994"/>
        <nd ref="454857540" lon="-91.8267272" lat="32.9146413"/>
        <nd ref="454857489" lon="-91.8273931" lat="32.9141368"/>
        <nd ref="454857490" lon="-91.827587" lat="32.9140565"/>
        <nd ref="454857486" lon="-91.8279656" lat="32.9140587"/>
        <nd ref="454857488" lon="-91.8281974" lat="32.9142344"/>
        <nd ref="454857512" lon="-91.8282892" lat="32.9150568"/>
        <nd ref="454857510" lon="-91.8284009" lat="32.9152592"/>
        <nd ref="454857511" lon="-91.8287139" lat="32.915457"/>
        <nd ref="454857517" lon="-91.8294986" lat="32.9156051"/>
        <nd ref="454857513" lon="-91.8304634" lat="32.9156661"/>
        <nd ref="454857520" lon="-91.8308118" lat="32.9155745"/>
        <nd ref="454857525" lon="-91.8310022" lat="32.9154666"/>
        <nd ref="454857518" lon="-91.8311338" lat="32.9151663"/>
        <nd ref="454857519" lon="-91.8311386" lat="32.9148003"/>
        <nd ref="454857071" lon="-91.8308713" lat="32.9144892"/>
        <nd ref="454857070" lon="-91.8299124" lat="32.9140459"/>
        <nd ref="454857073" lon="-91.8296109" lat="32.9137352"/>
        <nd ref="454857072" lon="-91.8291564" lat="32.9128002"/>
        <nd ref="454857066" lon="-91.8290288" lat="32.9123862"/>
        <nd ref="454857061" lon="-91.8289982" lat="32.9119205"/>
        <nd ref="454857068" lon="-91.8291115" lat="32.9112653"/>
        <nd ref="454857069" lon="-91.8292632" lat="32.9108421"/>
        <nd ref="454857067" lon="-91.829602" lat="32.9103248"/>
        <nd ref="454857081" lon="-91.8300439" lat="32.9098517"/>
        <nd ref="454857080" lon="-91.8302493" lat="32.9094289"/>
        <nd ref="454857078" lon="-91.8303465" lat="32.909127"/>
        <nd ref="454857079" lon="-91.8306953" lat="32.9072133"/>
        <nd ref="454857088" lon="-91.8309253" lat="32.9069173"/>
        <nd ref="454857089" lon="-91.8310893" lat="32.9065384"/>
        <nd ref="454857086" lon="-91.8311193" lat="32.9060004"/>
        <nd ref="454857087" lon="-91.8312203" lat="32.9055754"/>
        <nd ref="454857084" lon="-91.8313963" lat="32.9052524"/>
        <nd ref="454857085" lon="-91.8317543" lat="32.9048314"/>
        <nd ref="454857082" lon="-91.8341375" lat="32.9048633"/>
        <nd ref="454857083" lon="-91.8340975" lat="32.9079131"/>
        <nd ref="454857075" lon="-91.8381344" lat="32.9050194"/>
        <nd ref="454857074" lon="-91.8424715" lat="32.9020466"/>
        <nd ref="454857545" lon="-91.8443966" lat="32.9020186"/>
        <tag k="boundary" v="protected_area"/>
        <tag k="ele" v="32"/>
        <tag k="gnis:feature_id" v="533894"/>
        <tag k="leisure" v="park"/>
        <tag k="name" v="Chemin-A-Haut State Park"/>
        <tag k="operator" v="Louisiana Office of State Parks"/>
        <tag k="operator:type" v="public"/>
        <tag k="operator:wikidata" v="Q109319012"/>
        <tag k="wikidata" v="Q5090528"/>
        <tag k="wikipedia" v="en:Chemin-A-Haut State Park"/>
      </way>
    </old>
    <new>
      <way id="38476787" version="8" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420">
        <nd ref="454857545" lon="-91.8443966" lat="32.9020186"/>
        <nd ref="454857543" lon="-91.8448456" lat="32.9029955"/>
        <nd ref="454857544" lon="-91.8454046" lat="32.9035615"/>
        <nd ref="454857558" lon="-91.8468446" lat="32.9045575"/>
        <nd ref="454857559" lon="-91.8484107" lat="32.9049405"/>
        <nd ref="454857556" lon="-91.8504527" lat="32.9051045"/>
        <nd ref="454857557" lon="-91.8526598" lat="32.9046215"/>
        <nd ref="454857554" lon="-91.8539718" lat="32.9032796"/>
        <nd ref="454857555" lon="-91.8543438" lat="32.9026796"/>
        <nd ref="454857549" lon="-91.861496" lat="32.9027786"/>
        <nd ref="454857553" lon="-91.860905" lat="32.9054005"/>
        <nd ref="454857173" lon="-91.8561459" lat="32.9054995"/>
        <nd ref="454857184" lon="-91.8512167" lat="32.9054401"/>
        <nd ref="8643612207" lon="-91.8511352" lat="32.911748"/>
        <nd ref="8663693250" lon="-91.8511344" lat="32.9118078"/>
        <nd ref="8663693217" lon="-91.8511317" lat="32.9120205"/>
        <nd ref="8643612206" lon="-91.8511293" lat="32.9122029"/>
        <nd ref="454857160" lon="-91.8510753" lat="32.9163836"/>
        <nd ref="8663693232" lon="-91.8414184" lat="32.9165001"/>
        <nd ref="8663693255" lon="-91.8411104" lat="32.9165038"/>
        <nd ref="454857285" lon="-91.8399576" lat="32.9165177"/>
        <nd ref="8663693251" lon="-91.8399464" lat="32.9153915"/>
        <nd ref="8663693256" lon="-91.8399447" lat="32.9152154"/>
        <nd ref="454857292" lon="-91.8398865" lat="32.9093633"/>
        <nd ref="454857197" lon="-91.8341179" lat="32.9093928"/>
        <nd ref="454857275" lon="-91.8340808" lat="32.9165578"/>
        <nd ref="454857129" lon="-91.8338187" lat="32.9238093"/>
        <nd ref="454857135" lon="-91.8353657" lat="32.9238119"/>
        <nd ref="454857109" lon="-91.8349009" lat="32.9242902"/>
        <nd ref="454857117" lon="-91.8347323" lat="32.9246368"/>
        <nd ref="454857150" lon="-91.834218" lat="32.9251418"/>
        <nd ref="454857151" lon="-91.8341129" lat="32.9255159"/>
        <nd ref="454857147" lon="-91.833249" lat="32.9267538"/>
        <nd ref="454857148" lon="-91.832443" lat="32.9270364"/>
        <nd ref="454857090" lon="-91.8319378" lat="32.9267825"/>
        <nd ref="454857094" lon="-91.8314162" lat="32.9268389"/>
        <nd ref="454857332" lon="-91.8300767" lat="32.9274146"/>
        <nd ref="454857322" lon="-91.8292975" lat="32.9275448"/>
        <nd ref="454857295" lon="-91.8285996" lat="32.9275833"/>
        <nd ref="454857293" lon="-91.8255075" lat="32.9276153"/>
        <nd ref="454857316" lon="-91.8249224" lat="32.9276149"/>
        <nd ref="454857476" lon="-91.8243566" lat="32.9277079"/>
        <nd ref="454857480" lon="-91.8231934" lat="32.92824"/>
        <nd ref="454857479" lon="-91.8229331" lat="32.9279145"/>
        <nd ref="454857482" lon="-91.8222509" lat="32.9273068"/>
        <nd ref="454857481" lon="-91.8220956" lat="32.9270667"/>
        <nd ref="454857483" lon="-91.8210908" lat="32.9251771"/>
        <nd ref="454857485" lon="-91.820938" lat="32.9247557"/>
        <nd ref="454857343" lon="-91.820805" lat="32.9245445"/>
        <nd ref="454857351" lon="-91.8207085" lat="32.9241427"/>
        <nd ref="454857387" lon="-91.8207496" lat="32.9237505"/>
        <nd ref="454857368" lon="-91.8208294" lat="32.9236686"/>
        <nd ref="454857403" lon="-91.8252572" lat="32.9236756"/>
        <nd ref="454857421" lon="-91.8252823" lat="32.9226449"/>
        <nd ref="454857444" lon="-91.8256608" lat="32.9226625"/>
        <nd ref="454857435" lon="-91.8258778" lat="32.9226137"/>
        <nd ref="454857457" lon="-91.8263411" lat="32.9223782"/>
        <nd ref="454857470" lon="-91.8271573" lat="32.9214707"/>
        <nd ref="454857471" lon="-91.8278228" lat="32.9208158"/>
        <nd ref="454857472" lon="-91.8279871" lat="32.9205835"/>
        <nd ref="454857473" lon="-91.8281948" lat="32.9198528"/>
        <nd ref="454857474" lon="-91.8281165" lat="32.9194017"/>
        <nd ref="454857475" lon="-91.827807" lat="32.9190067"/>
        <nd ref="454857532" lon="-91.8270937" lat="32.9185523"/>
        <nd ref="454857530" lon="-91.8267751" lat="32.9182978"/>
        <nd ref="454857535" lon="-91.8264526" lat="32.9178034"/>
        <nd ref="454857534" lon="-91.8264445" lat="32.917448"/>
        <nd ref="454857533" lon="-91.826539" lat="32.9171047"/>
        <nd ref="454857539" lon="-91.8265842" lat="32.9165342"/>
        <nd ref="454857538" lon="-91.8262383" lat="32.9156588"/>
        <nd ref="454857542" lon="-91.8262413" lat="32.9154327"/>
        <nd ref="454857541" lon="-91.8264226" lat="32.914994"/>
        <nd ref="454857540" lon="-91.8267272" lat="32.9146413"/>
        <nd ref="454857489" lon="-91.8273931" lat="32.9141368"/>
        <nd ref="454857490" lon="-91.827587" lat="32.9140565"/>
        <nd ref="454857486" lon="-91.8279656" lat="32.9140587"/>
        <nd ref="454857488" lon="-91.8281974" lat="32.9142344"/>
        <nd ref="454857512" lon="-91.8282892" lat="32.9150568"/>
        <nd ref="454857510" lon="-91.8284009" lat="32.9152592"/>
        <nd ref="454857511" lon="-91.8287139" lat="32.915457"/>
        <nd ref="454857517" lon="-91.8294986" lat="32.9156051"/>
        <nd ref="454857513" lon="-91.8304634" lat="32.9156661"/>
        <nd ref="454857520" lon="-91.8308118" lat="32.9155745"/>
        <nd ref="454857525" lon="-91.8310022" lat="32.9154666"/>
        <nd ref="454857518" lon="-91.8311338" lat="32.9151663"/>
        <nd ref="454857519" lon="-91.8311386" lat="32.9148003"/>
        <nd ref="454857071" lon="-91.8308713" lat="32.9144892"/>
        <nd ref="454857070" lon="-91.8299124" lat="32.9140459"/>
        <nd ref="454857073" lon="-91.8296109" lat="32.9137352"/>
        <nd ref="454857072" lon="-91.8291564" lat="32.9128002"/>
        <nd ref="454857066" lon="-91.8290288" lat="32.9123862"/>
        <nd ref="454857061" lon="-91.8289982" lat="32.9119205"/>
        <nd ref="454857068" lon="-91.8291115" lat="32.9112653"/>
        <nd ref="454857069" lon="-91.8292632" lat="32.9108421"/>
        <nd ref="454857067" lon="-91.829602" lat="32.9103248"/>
        <nd ref="454857081" lon="-91.8300439" lat="32.9098517"/>
        <nd ref="454857080" lon="-91.8302493" lat="32.9094289"/>
        <nd ref="454857078" lon="-91.8303465" lat="32.909127"/>
        <nd ref="454857079" lon="-91.8306953" lat="32.9072133"/>
        <nd ref="454857088" lon="-91.8309253" lat="32.9069173"/>
        <nd ref="454857089" lon="-91.8310893" lat="32.9065384"/>
        <nd ref="454857086" lon="-91.8311193" lat="32.9060004"/>
        <nd ref="454857087" lon="-91.8312203" lat="32.9055754"/>
        <nd ref="454857084" lon="-91.8313963" lat="32.9052524"/>
        <nd ref="454857085" lon="-91.8317543" lat="32.9048314"/>
        <nd ref="454857082" lon="-91.8341375" lat="32.9048633"/>
        <nd ref="454857083" lon="-91.8340975" lat="32.9079131"/>
        <nd ref="454857075" lon="-91.8381344" lat="32.9050194"/>
        <nd ref="454857074" lon="-91.8424715" lat="32.9020466"/>
        <nd ref="454857545" lon="-91.8443966" lat="32.9020186"/>
        <tag k="boundary" v="protected_area"/>
        <tag k="ele" v="32"/>
        <tag k="gnis:feature_id" v="533894"/>
        <tag k="leisure" v="nature_reserve"/>
        <tag k="name" v="Chemin-A-Haut State Park"/>
        <tag k="operator" v="Louisiana Office of State Parks"/>
        <tag k="operator:type" v="government"/>
        <tag k="operator:wikidata" v="Q109319012"/>
        <tag k="protection_title" v="State Park"/>
        <tag k="website" v="https://www.lastateparks.com/parks-preserves/chemin-a-haut-state-park"/>
        <tag k="wikidata" v="Q5090528"/>
        <tag k="wikipedia" v="en:Chemin-A-Haut State Park"/>
      </way>
    </new>
  </action>
  <action type="modify">
    <old>
      <way id="758279917" version="1" user="MatthewAndersonUS80" uid="818552" timestamp="2019-12-24T04:29:30Z" changeset="78799793">
        <bounds minlat="32.9120923" minlon="-91.8507376" maxlat="32.930614" maxlon="-91.837942"/>
        <nd ref="115828379" lon="-91.8507376" lat="32.9120923"/>
        <nd ref="115868091" lon="-91.849971" lat="32.912256"/>
        <nd ref="115868096" lon="-91.842096" lat="32.91558"/>
        <nd ref="115868099" lon="-91.841579" lat="32.915915"/>
        <nd ref="115839045" lon="-91.84139" lat="32.916038"/>
        <nd ref="115868104" lon="-91.840304" lat="32.918559"/>
        <nd ref="115868106" lon="-91.840005" lat="32.91904"/>
        <nd ref="115868109" lon="-91.838105" lat="32.923462"/>
        <nd ref="115868112" lon="-91.837942" lat="32.924057"/>
        <nd ref="115837223" lon="-91.837942" lat="32.926234"/>
        <nd ref="115868115" lon="-91.837958" lat="32.927515"/>
        <nd ref="115849652" lon="-91.837997" lat="32.930614"/>
        <tag k="highway" v="residential"/>
        <tag k="name" v="Loop Park Road"/>
        <tag k="tiger:cfcc" v="A41"/>
        <tag k="tiger:county" v="Morehouse, LA"/>
        <tag k="tiger:name_base" v="Loop Park"/>
        <tag k="tiger:name_type" v="Rd"/>
        <tag k="tiger:reviewed" v="no"/>
        <tag k="tiger:zip_left" v="71220"/>
        <tag k="tiger:zip_right" v="71220"/>
      </way>
    </old>
    <new>
      <way id="758279917" version="2" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420">
        <nd ref="115828379" lon="-91.8507376" lat="32.9120923"/>
        <nd ref="115868096" lon="-91.8425037" lat="32.9155243"/>
        <nd ref="12550908584" lon="-91.8420375" lat="32.9157457"/>
        <nd ref="12550908585" lon="-91.8416835" lat="32.9159882"/>
        <nd ref="115839045" lon="-91.8415109" lat="32.9161344"/>
        <nd ref="115868104" lon="-91.840304" lat="32.918559"/>
        <nd ref="115868106" lon="-91.840005" lat="32.91904"/>
        <nd ref="115868109" lon="-91.838105" lat="32.923462"/>
        <nd ref="115868112" lon="-91.837942" lat="32.924057"/>
        <nd ref="115837223" lon="-91.837942" lat="32.926234"/>
        <nd ref="115868115" lon="-91.837958" lat="32.927515"/>
        <nd ref="115849652" lon="-91.837997" lat="32.930614"/>
        <tag k="highway" v="residential"/>
        <tag k="name" v="Loop Park Road"/>
        <tag k="tiger:cfcc" v="A41"/>
        <tag k="tiger:county" v="Morehouse, LA"/>
        <tag k="tiger:name_base" v="Loop Park"/>
        <tag k="tiger:name_type" v="Rd"/>
        <tag k="tiger:reviewed" v="no"/>
        <tag k="tiger:zip_left" v="71220"/>
        <tag k="tiger:zip_right" v="71220"/>
      </way>
    </new>
  </action>
  <action type="delete">
    <old>
      <relation id="12600902" version="3" user="zluuzki" uid="12785610" timestamp="2023-08-13T09:15:33Z" changeset="139814967">
        <bounds minlat="32.9020186" minlon="-91.861496" maxlat="32.92824" maxlon="-91.8207085"/>
        <member ref="38476787" role="outer" type="way">
          <nd lon="-91.8443966" lat="32.9020186"/>
          <nd lon="-91.8448456" lat="32.9029955"/>
          <nd lon="-91.8454046" lat="32.9035615"/>
          <nd lon="-91.8468446" lat="32.9045575"/>
          <nd lon="-91.8484107" lat="32.9049405"/>
          <nd lon="-91.8504527" lat="32.9051045"/>
          <nd lon="-91.8526598" lat="32.9046215"/>
          <nd lon="-91.8539718" lat="32.9032796"/>
          <nd lon="-91.8543438" lat="32.9026796"/>
          <nd lon="-91.861496" lat="32.9027786"/>
          <nd lon="-91.860905" lat="32.9054005"/>
          <nd lon="-91.8561459" lat="32.9054995"/>
          <nd lon="-91.8512167" lat="32.9054401"/>
          <nd lon="-91.8511352" lat="32.911748"/>
          <nd lon="-91.8511344" lat="32.9118078"/>
          <nd lon="-91.8511317" lat="32.9120205"/>
          <nd lon="-91.8511293" lat="32.9122029"/>
          <nd lon="-91.8510753" lat="32.9163836"/>
          <nd lon="-91.8414184" lat="32.9165001"/>
          <nd lon="-91.8411104" lat="32.9165038"/>
          <nd lon="-91.8399576" lat="32.9165177"/>
          <nd lon="-91.8399464" lat="32.9153915"/>
          <nd lon="-91.8399447" lat="32.9152154"/>
          <nd lon="-91.8398865" lat="32.9093633"/>
          <nd lon="-91.8341179" lat="32.9093928"/>
          <nd lon="-91.8340808" lat="32.9165578"/>
          <nd lon="-91.8338187" lat="32.9238093"/>
          <nd lon="-91.8353657" lat="32.9238119"/>
          <nd lon="-91.8349009" lat="32.9242902"/>
          <nd lon="-91.8347323" lat="32.9246368"/>
          <nd lon="-91.834218" lat="32.9251418"/>
          <nd lon="-91.8341129" lat="32.9255159"/>
          <nd lon="-91.833249" lat="32.9267538"/>
          <nd lon="-91.832443" lat="32.9270364"/>
          <nd lon="-91.8319378" lat="32.9267825"/>
          <nd lon="-91.8314162" lat="32.9268389"/>
          <nd lon="-91.8300767" lat="32.9274146"/>
          <nd lon="-91.8292975" lat="32.9275448"/>
          <nd lon="-91.8285996" lat="32.9275833"/>
          <nd lon="-91.8255075" lat="32.9276153"/>
          <nd lon="-91.8249224" lat="32.9276149"/>
          <nd lon="-91.8243566" lat="32.9277079"/>
          <nd lon="-91.8231934" lat="32.92824"/>
          <nd lon="-91.8229331" lat="32.9279145"/>
          <nd lon="-91.8222509" lat="32.9273068"/>
          <nd lon="-91.8220956" lat="32.9270667"/>
          <nd lon="-91.8210908" lat="32.9251771"/>
          <nd lon="-91.820938" lat="32.9247557"/>
          <nd lon="-91.820805" lat="32.9245445"/>
          <nd lon="-91.8207085" lat="32.9241427"/>
          <nd lon="-91.8207496" lat="32.9237505"/>
          <nd lon="-91.8208294" lat="32.9236686"/>
          <nd lon="-91.8252572" lat="32.9236756"/>
          <nd lon="-91.8252823" lat="32.9226449"/>
          <nd lon="-91.8256608" lat="32.9226625"/>
          <nd lon="-91.8258778" lat="32.9226137"/>
          <nd lon="-91.8263411" lat="32.9223782"/>
          <nd lon="-91.8271573" lat="32.9214707"/>
          <nd lon="-91.8278228" lat="32.9208158"/>
          <nd lon="-91.8279871" lat="32.9205835"/>
          <nd lon="-91.8281948" lat="32.9198528"/>
          <nd lon="-91.8281165" lat="32.9194017"/>
          <nd lon="-91.827807" lat="32.9190067"/>
          <nd lon="-91.8270937" lat="32.9185523"/>
          <nd lon="-91.8267751" lat="32.9182978"/>
          <nd lon="-91.8264526" lat="32.9178034"/>
          <nd lon="-91.8264445" lat="32.917448"/>
          <nd lon="-91.826539" lat="32.9171047"/>
          <nd lon="-91.8265842" lat="32.9165342"/>
          <nd lon="-91.8262383" lat="32.9156588"/>
          <nd lon="-91.8262413" lat="32.9154327"/>
          <nd lon="-91.8264226" lat="32.914994"/>
          <nd lon="-91.8267272" lat="32.9146413"/>
          <nd lon="-91.8273931" lat="32.9141368"/>
          <nd lon="-91.827587" lat="32.9140565"/>
          <nd lon="-91.8279656" lat="32.9140587"/>
          <nd lon="-91.8281974" lat="32.9142344"/>
          <nd lon="-91.8282892" lat="32.9150568"/>
          <nd lon="-91.8284009" lat="32.9152592"/>
          <nd lon="-91.8287139" lat="32.915457"/>
          <nd lon="-91.8294986" lat="32.9156051"/>
          <nd lon="-91.8304634" lat="32.9156661"/>
          <nd lon="-91.8308118" lat="32.9155745"/>
          <nd lon="-91.8310022" lat="32.9154666"/>
          <nd lon="-91.8311338" lat="32.9151663"/>
          <nd lon="-91.8311386" lat="32.9148003"/>
          <nd lon="-91.8308713" lat="32.9144892"/>
          <nd lon="-91.8299124" lat="32.9140459"/>
          <nd lon="-91.8296109" lat="32.9137352"/>
          <nd lon="-91.8291564" lat="32.9128002"/>
          <nd lon="-91.8290288" lat="32.9123862"/>
          <nd lon="-91.8289982" lat="32.9119205"/>
          <nd lon="-91.8291115" lat="32.9112653"/>
          <nd lon="-91.8292632" lat="32.9108421"/>
          <nd lon="-91.829602" lat="32.9103248"/>
          <nd lon="-91.8300439" lat="32.9098517"/>
          <nd lon="-91.8302493" lat="32.9094289"/>
          <nd lon="-91.8303465" lat="32.909127"/>
          <nd lon="-91.8306953" lat="32.9072133"/>
          <nd lon="-91.8309253" lat="32.9069173"/>
          <nd lon="-91.8310893" lat="32.9065384"/>
          <nd lon="-91.8311193" lat="32.9060004"/>
          <nd lon="-91.8312203" lat="32.9055754"/>
          <nd lon="-91.8313963" lat="32.9052524"/>
          <nd lon="-91.8317543" lat="32.9048314"/>
          <nd lon="-91.8341375" lat="32.9048633"/>
          <nd lon="-91.8340975" lat="32.9079131"/>
          <nd lon="-91.8381344" lat="32.9050194"/>
          <nd lon="-91.8424715" lat="32.9020466"/>
          <nd lon="-91.8443966" lat="32.9020186"/>
        </member>
        <tag k="leisure" v="nature_reserve"/>
        <tag k="name" v="Chemin-A-Haut State Park"/>
        <tag k="type" v="multipolygon"/>
      </relation>
    </old>
    <new>
      <relation id="12600902" version="4" timestamp="2025-02-02T09:05:06Z" uid="8794039" user="jake-low" changeset="162037420" visible="false"/>
    </new>
  </action>
</osm>
