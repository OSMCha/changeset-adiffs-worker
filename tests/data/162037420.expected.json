{
  "highway": [
    "residential"
  ],
  "protection_title": [
    "State Park"
  ],
  "website": [
    "https://www.lastateparks.com/parks-preserves/chemin-a-haut-state-park"
  ],
  "leisure": [
    "nature_reserve",
    "park"
  ],
  "operator:type": [
    "government",
    "public"
  ],
  "boundary": [
    "protected_area"
  ]
}
